### ISSUE TITLE
Import of paypal transaction fails

### ISSUE THREAD

**Issue by jottr** [johannesgerer/buchhaltung#21]
I'm not sure if this is an issue with my shell configuration or rather an issue with how `buchhaltung` parses Umlauts.  

```
$ buchhaltung import /tmp/paypal.csv paypal jottr
buchhaltung: Headers do not match. Expected by format 'paypal' version '2016':
["Datum"," Zeit"," Zeitzone"," Name"," Typ"," Status"," W\228hrung"," Brutto"," Geb\252hr"," Netto"," Von E-Mail-Adresse"," An E-Mail-Adresse"," Transactionscode"," Status der Gegenpartei","
Adressstatus"," Artikelbezeichnung"," Artikelnummer"," Betrag f\252r Versandkosten"," Versicherungsbetrag"," Umsatzsteuer"," Option 1 - Name"," Option 1 - Wert"," Option 2 - Name"," Option 2 - Wert"," Auktions-Site"," K\228ufer-ID"," Artikel-URL"," Angebotsende"," Vorgangs-Nr."," Rechnungs-Nr."," Txn-Referenzkennung"," Rechnungsnummer"," Individuelle Nummer"," Belegnummer"," Guthaben"," Adresszeile 1"," Zus\228tzliche Angaben"," Ort"," Staat/Provinz/Region/Landkreis/Territorium/Pr\228fektur/Republik"," PLZ"," Land"," Telefonnummer"," "]
Given:
["Datum"," Zeit"," Zeitzone"," Name"," Typ"," Status"," W\195\164hrung"," Brutto"," Geb\195\188hr"," Netto"," Von E-Mail-Adresse"," An E-Mail-Adresse"," Transactionscode"," Zahlungsart","Status der Gegenpartei"," Adressstatus"," Artikelbezeichnung"," Artikelnummer"," Betrag f\195\188r Versandkosten"," Versicherungsbetrag"," Umsatzsteuer"," Option 1 - Name"," Option 1 - Wert"," Option 2 - Name"," Option 2 - Wert"," Auktions-Site"," K\195\164ufer-ID"," Artikel-URL"," Angebotsende"," Txn-Referenzkennung"," Rechnungsnummer"," Individuelle Nummer"," Belegnummer"," Guthaben"," Adresszeile 1"," Zus\195\164tzliche Angaben"," Ort"," Staat/Provinz/Region/Landkreis/Territorium/Pr\195\164fektur/Republik"," PLZ"," Land"," Telefonnummer"," "]

CallStack (from HasCallStack):
  error, called at src/Buchhaltung/Commandline.hs:37:11 in main:Buchhaltung.Commandline
```

**Comment by johannesgerer** [comment]
This yells you that the header you supplied is different from what it expects for the 2016 version of PayPal. So either try selecting a different version when importing or if none matches add this new format to the source code.

**Comment by jottr** [comment]
Not sure where you found a 2016 version of the transaction history.
The most recent revision seems to be from 2013:  

![bildschirmfoto 2017-10-05 um 23 04 18](https://user-images.githubusercontent.com/2744198/31252486-a34fcd80-aa21-11e7-8860-9a3eaba8d326.png)

Viewing above [changelog](https://www.paypal.com/de/cgi-bin/webscr?cmd=p/acc/history-download-messages) requires login to paypal, hence the screenshot.

Anyways, even after exporting "Alle Aktivitäten" to csv, the required field `Vorgangs-Nr.` is not present.

**Comment by johannesgerer** [comment]
I don't use there normative list but rather what ever they give me in the data and that keeps changing. Just add your current version to the source. (Or I can if you give me some time.)
But make sure that you used the right settings to produce it. See buchhaltung import -h

**Comment by johannesgerer** [comment]
solved by #22?
